.doc_gen/metadata/redshift_metadata.yaml

```diff
@@ -78,7 +95,7 @@ redshift_DeleteCluster:
 redshift_DescribeClusters:
   title: Describe an &RS; cluster using an &AWS; SDK
   title_abbrev: Describe your clusters
```

Comment:
Suggested: "Describe clusters" for the title, and "describe &RS; clusters." for the synopsis.

Reply:
addressed